programmers/비밀지도.js

```diff
@@ -0,0 +1,25 @@
+function solution(n, arr1, arr2) {
+ 
+    let answer = [];
+    
+    for (let i = 0; i < arr1.length; i++) {
+        let bin = (arr1[i] | arr2[i]).toString(2);
+        
+        if (bin.length < n) {
+            let s = n - bin.length;
+            let t = '';
+            for (let k = 0; k < s; k++) {
+                t += 0;
+            }
+            bin = t + bin;
+        }
+        
```

Comment:
```suggestion
        while (bin.length < n) bin = 0 + bin
```
이렇게 한 줄로도 할 수 있을 거 같아요